release_notes/2.2.3.md

```diff
@@ -3,6 +3,5 @@
 
 **Version 2.2.3 - Released January 27, 2022**
```

Comment:
this version doesn't exist on https://splunkbase.splunk.com/app/5875/

should we just remove this whole section and move the applicable points to unreleased?

Reply:
oh I see you did that in the main PR https://github.com/splunk-soar-connectors/winrm/pull/11/files

Reply:
although in that case we should delete the headers on the 2.2.3 md notes because they're incorrect anyways (and no longer necessary)